### PR TITLE
chore: Create chart docsite to enable in v9 docs

### PR DESCRIPTION
**Pull request by AtishayMsft**
## New Behavior

Create a new docsite for chart controls.
This will be enabled in fluent v9 docsite using storybook composition.

## Related Issue(s)

- Fixes #

### CODE REVIEW

File: apps/pr-deploy-site/just.config.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ const dependencies = [
   '@fluentui/public-docsite',
```
Comment (fabricteam) [inline apps/pr-deploy-site/just.config.ts:24]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33283&commitId=5878f4ac91313fd901df4405271e20564f6edb38&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|17| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|3| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|27| Changed|
|Drawer.overlay drawer full.chromium.png|3320| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Comment (AtishayMsft) [reply apps/pr-deploy-site/just.config.ts:24]: Unrelated screenshots.

**Pull request merged**